config/makeCoaddTempExp_goodSeeing.py

```diff
@@ -0,0 +1,22 @@
+from lsst.decam.hits import MaxPsfWcsSelectImagesTask
+
+config.bgSubtracted=True
```

Comment:
`config` is never defined; how does this not cause a `NameError`?

Reply:
I don't know, but I think it's handled by CommandLineTask?  See https://pipelines.lsst.io/v/DM-11597/modules/lsst.pipe.base/command-line-task-config-howto.html